sopel/modules/clock.py

```diff
@@ -222,7 +222,7 @@ def get_user_format(bot, trigger):
 
 @module.commands('setchanneltz', 'setctz')
 @module.example('.setctz America/New_York')
-@module.require_privilege(module.OP)
+@module.require_privilege(module.OP, message='You must be opped up to set the channel timezone.')
```

Comment:
An idea for perusal by interested parties from our IRC:

```suggestion
@module.require_privilege(module.OP, message='Changing the channel timezone requires OP privileges.')
```